test/test_scenario/test_scenario.py

```diff
@@ -322,23 +293,16 @@ def test_str_cast_instances(self):
         self.assertIsInstance(self.scen.train_insts[0], str)
         self.assertIsInstance(self.scen.train_insts[1], str)
 
-    def test_create_scenario_option_for_doc(self):
```

Comment:
It appears that this crucial test is removed.

Reply:
now part of the unit test of the cmdreader